### PR TITLE
docs: more snapping fixes

### PR DESCRIPTION
**Pull request by mattklein123**

### CODE REVIEW

**Comment by mattklein123** [comment]
@bndw @moderation

File: docs/root/configuration/secret.rst [hunk 1]
```diff
@@ -20,8 +20,8 @@ The connection between Envoy proxy and SDS server has to be secure. One option i
 SDS server
 ----------
 
-A SDS server needs to implement the gRPC service `SecretDiscoveryService <https://github.com/envoyproxy/envoy/blob/master/api/envoy/service/discovery/v2/sds.proto>`_.
-It follows the same protocol as other `xDS <https://github.com/envoyproxy/data-plane-api/blob/master/XDS_PROTOCOL.md>`_
+A SDS server needs to implement the gRPC service :repo:`SecretDiscoveryService </api/envoy/service/discovery/v2/sds.proto>`.
```
Comment (venilnoronha) [inline docs/root/configuration/secret.rst]: nit: drop the / prefix in the URI.

**Comment by moderation** [comment]
LGTM

**Pull request merged**